diff --git a/node_modules/workbox-build/build/lib/module-registry.js b/node_modules/workbox-build/build/lib/module-registry.js
index 9a34dd9..58b9617 100644
--- a/node_modules/workbox-build/build/lib/module-registry.js
+++ b/node_modules/workbox-build/build/lib/module-registry.js
@@ -40,8 +40,9 @@ class ModuleRegistry {
             // strip the '/package.json' from the resolved path.
             const pkgJsonPath = require.resolve(`${pkg}/package.json`);
             const pkgRoot = upath_1.default.dirname(pkgJsonPath);
+            const escapedPkgRoot = pkgRoot.replace(/'/g, "\\'");
             const importStatement = (0, common_tags_1.oneLine) `import {${moduleName} as ${localName}} from
-        '${pkgRoot}/${moduleName}.mjs';`;
+        '${escapedPkgRoot}/${moduleName}.mjs';`;
             workboxModuleImports.push(importStatement);
         }
         return workboxModuleImports;
